### PR TITLE
Part rst

### PR DESCRIPTION
**Pull request by davidsean**
Fixes #

Description of changes:
 - almost all python now
 - Konrad made images of most constraints
 - use same terms between part.rst and particledata
 - still need someone do go over virtualsites

PR Checklist
------------
 - [ ] Tests?
   - [ ] Interface
   - [ ] Core 
 - [ ] Docs?

### CODE REVIEW

**Comment by KaiSzuttor** [comment]
There are a lot of unnecessary empty lines in the rst file.

**Comment by fweik** [comment]
Could you please do a merge ignoring white space conflicts, e.g.
```
git merge -s recursive -Xignore-space-change
```
or resolve the 41 conflicts by hand.

File: src/python/espressomd/particle_data.pyx [hunk 1]
```diff
@@ -1022,11 +961,7 @@ cdef class ParticleHandle(object):
                     """
                     The particle translational frictional coefficient used in the Langevin thermostat.
 
-                    gamma_rot : list of floats
-
-                    .. note::
-
-                       This needs the feature LANGEVIN_PER_PARTICLE, ROTATION and ROTATIONAL_INERTIA
+                    gamma_rot : list of floats (if using feature ROTATIONAL_INTERIA)
```
Comment (KaiSzuttor) [inline src/python/espressomd/particle_data.pyx]: The feature ROTATIONAL_INERTIA is required anyways

Comment (davidsean) [reply src/python/espressomd/particle_data.pyx]: I was trying to differentiate between the float gamma_rot and the vector, which depends on the ROTATIONAL_INERTIA feature. I guess  the documentation build will take care of that?

**Comment by KaiSzuttor** [comment]
The still seems to be an error in cython, please check @davidsean

**Review by fweik — APPROVED** [review]

**Pull request merged**